package.json

```diff
@@ -10,28 +10,29 @@
     "hasher": "^1.2.0"
   },
   "devDependencies": {
-    "@types/chai": "^3.4.28",
-    "@types/hasher": "0.0.27",
-    "@types/crossroads": "0.0.28",
-    "@types/jasmine": "2.8.9",
-    "@types/jasmine-ajax": "^3.1.27",
-    "@types/requirejs": "^2.1.26",
     "@syncfusion/ej2-base": "*",
```

Comment:
Why so many unnecessary changes, Add only the required packages and don't change the existing order